app/views/katello/api/v2/subscription_facet/subscription_status.json.rabl

```diff
@@ -0,0 +1,3 @@
+
+attributes :to_global => :color_code
```

Comment:
As an API parameter, to_global and to_label seem strange to me. I am not sure how to interpret them.